src/scheduler/random.rs

```diff
@@ -58,7 +58,23 @@ impl RandomScheduler {
     /// Two RandomSchedulers initialized with the same seed will make the same scheduling decisions
```

Comment:
Need to update the comment to explain the effect of setting `SHUTTLE_RANDOM_SEED`